llvm/lib/Transforms/Vectorize/VPlan.cpp

```diff
@@ -823,16 +821,25 @@ void VPRegionBlock::print(raw_ostream &O, const Twine &Indent,
 }
 #endif
 
+VPlan::VPlan(VPBasicBlock *Preheader, VPValue *TC, VPBasicBlock *Entry,
```

Comment:
Perhaps some renaming can help be more consistent yet support gradual transition, e.g.,
```suggestion
VPlan::VPlan(VPBasicBlock *OriginalPreheader, VPValue *TC, VPBasicBlock *EntryVectorPreHeader,
```
?